test/functional/content_view/version/incremental_update_test.rb

```diff
@@ -88,4 +88,21 @@
     result = run_cmd(@cmd + params)
     assert_equal(result.exit_code, 0)
   end
+
+  it "performs incremental update with packages" do
+    params = ['--content-view-version-id=5', '--packages=bla', '--lifecycle-environment-ids=1']
+
+    expect_generic_search(
+      :packages, params: {search: "name = \"bla\""}, returns: {'id' => 5}
+    )
+
+    ex = api_expects(:content_view_versions, :incremental_update, 'Incremental Update') do |par|
+      par[:content_view_version_environments][0][:content_view_version_id] == 5 &&
+        par["add_content"]["package_ids"] == [5]
+    end
+    ex.returns(index_response([{'id' => 5}]), 'id' => '3', 'state' => 'stopped')
```

Comment:
The index_reponse should be on line 96, not here. You only need the `'id' => '3'` here.